### PR TITLE
Update help menu

### PR DESCRIPTION
**Pull request by Froloket64**
Updates the help menu with respect to #46 and also adds <kbd>j</kbd> and <kbd>k</kbd> key annotations

### CODE REVIEW

File: src/component/task_list.rs [hunk 1]
```diff
@@ -59,12 +56,17 @@ impl TaskList {
...
                 "Moves the task up on the task list",
             ),
+            HelpAction::new(
+                KeyCode::Char('h'),
+                "h",
+                "Gives selected task lower priority",
+            ),
             HelpAction::new(KeyCode::Char('t'), "t", "Add tags to the task"),
-            HelpAction::new(KeyCode::Char('s'), "s", "Toggle task sort by priority"),
+            HelpAction::new(KeyCode::Char('s'), "s", "Toggle task sort (by priority)"),
```
Comment (SleepySwords) [inline src/component/task_list.rs:65]: Just realised, this should probably just be `Sort tasks`

Comment (Froloket64) [reply src/component/task_list.rs:65]: Oh, good point)

Comment (Froloket64) [reply src/component/task_list.rs:65]: How about this?
```rust
HelpAction::new(KeyCode::Char('t'), "t", "Adds tags to the task"),
HelpAction::new(KeyCode::Char('s'), "s", "Sorts tasks (by priority)"),
HelpAction::new(
    KeyCode::Char('S'),
    "S",
    "Toggles automatic task sort",
),
```

Comment (SleepySwords) [reply src/component/task_list.rs:65]: Sounds good!

**Pull request merged**